### PR TITLE
ar71xx: add metadata to some TP-Link images, fix Archer C60 factory image

### PR DESCRIPTION
**Pull request by heinzek**
First commit add metadata to TP-Link sysupgrade images created by tplink-safeloader with "magic" 2705.

Second commit fix build factory firmware TP-Link Archer C60v1. Size of partition "SupportList" is only 256 bytes, and can contain only 3 entries.

### CODE REVIEW

File: target/linux/ar71xx/image/tp-link.mk [hunk 1]
```diff
@@ -121,7 +121,7 @@ endef
 define Device/archer-cxx
   KERNEL := kernel-bin | patch-cmdline | lzma | uImageArcher lzma
   IMAGES := sysupgrade.bin factory.bin
-  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade
+  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade | append-metadata
```
Comment (pepe2k) [inline target/linux/ar71xx/image/tp-link.mk]: `tplink-safeloader` checks if kernel and rootfs fit in defined partitions in image, but as you use `append-metadata` here as a last build step and appended metadata is not stripped from sysupgrade image before writing it to flash, `check-size $$$$(IMAGE_SIZE)` is needed here.

File: target/linux/ar71xx/image/tp-link.mk [hunk 2]
```diff
@@ -1093,8 +1097,9 @@ define Device/tl-wr942n-v1
   IMAGE_SIZE := 14464k
   KERNEL := kernel-bin | patch-cmdline | lzma | uImageArcher lzma
   IMAGES := sysupgrade.bin factory.bin
-  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade
+  IMAGE/sysupgrade.bin := append-rootfs | tplink-safeloader sysupgrade | append-metadata
```
Comment (pepe2k) [inline target/linux/ar71xx/image/tp-link.mk]: Same here, please add `check-size $$$$(IMAGE_SIZE)` as a last build step for sysupgrade image.

**Comment by heinzek** [comment]
done

**Comment by pepe2k** [comment]
@heinzek,

Thank you for your contribution! I have merged your changes into [my staging tree](https://git.lede-project.org/?p=lede/pepe2k/staging.git;a=summary).

Dzięki!

Cheers,
Piotr

**Pull request closed without merging**